src/components/v2v3/shared/SplitItem/SplitPercentValue.tsx

```diff
@@ -0,0 +1,27 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import { DiffedItem } from 'components/v2v3/shared/DiffedItem'
+import { formatSplitPercent } from 'utils/v2v3/math'
+
+export function SplitPercentValue({
+  percent,
+  oldPercent,
```

Comment:
@johnnyd-eth sorry if I wasnt clear before. My point was that I don't think any of this stuff in `SplitItem` directory should know anything about diffs. No diff props or anything. We can create DiffedSplitPercentValue component if needed and use where needed. This DiffedSplitPercentValue would likely use `SplitPercentValue` under the hood too.